--- DrizztSaga\drizztsaga.tp2	Mon May 25 10:36:23 2009
+++ C:\BWP Patchstudio\patched files\DrizztSaga\drizztsaga.tp2	Fri Apr 16 21:27:51 2010
@@ -251,10 +251,10 @@
 MKDIR ~DZTS-ARE~
 
 COPY + ~Setup-DRIZZTSAGA.exe~ ~WeiDU.exe~
-COPY + ~chitin.key~ ~DRIZZTSAGA/chitin.dzt~
-COPY_LARGE + ~dialog.tlk~ ~DRIZZTSAGA/dialog.dzt~
-MKDIR ~DRIZZTSAGA/over.bak~
-COPY_LARGE + ~override~ ~DRIZZTSAGA/over.bak~
+// COPY + ~chitin.key~ ~DRIZZTSAGA/chitin.dzt~
+// COPY_LARGE + ~dialog.tlk~ ~DRIZZTSAGA/dialog.dzt~
+// MKDIR ~DRIZZTSAGA/over.bak~
+// COPY_LARGE + ~override~ ~DRIZZTSAGA/over.bak~
 
 AT_UNINSTALL ~DRIZZTSAGA/Uninstall.bat~
 
