--- TS\TSAreasSounds.bat	Sat May 26 14:32:37 2007
+++ C:\BWP Patchstudio\patched files\TS\TSAreasSounds.bat	Tue Mar 23 10:33:54 2010
@@ -13,16 +13,16 @@
 tisunpack -s -h -o TS-TIS/TT0204.tis TS\TIZ/TT0204.tiz
 tisunpack -s -h -o TS-TIS/TT0301.tis TS\TIZ/TT0301.tiz
 tisunpack -s -h -o TS-TIS/TT0302.tis TS\TIZ/TT0302.tiz
-tisunpack -s -h -o TS-TIS/TT0303.tis TS\TIZ/TT0303.tiz
-tisunpack -s -h -o TS-TIS/TT0304.tis TS\TIZ/TT0304.tiz
-tisunpack -s -h -o TS-TIS/TT0305.tis TS\TIZ/TT0305.tiz
-tisunpack -s -h -o TS-TIS/TT0306.tis TS\TIZ/TT0306.tiz
-tisunpack -s -h -o TS-TIS/TT0307.tis TS\TIZ/TT0307.tiz
-tisunpack -s -h -o TS-TIS/TT0308.tis TS\TIZ/TT0308.tiz
+tisunpack -s -h -o TS-BAM/TT0303.tis TS\TIZ/TT0303.tiz
+tisunpack -s -h -o TS-BCS/TT0304.tis TS\TIZ/TT0304.tiz
+tisunpack -s -h -o TS-CRE/TT0305.tis TS\TIZ/TT0305.tiz
+tisunpack -s -h -o TS-DLG/TT0306.tis TS\TIZ/TT0306.tiz
+tisunpack -s -h -o TS-ITM/TT0307.tis TS\TIZ/TT0307.tiz
+tisunpack -s -h -o TS-RULE/TT0308.tis TS\TIZ/TT0308.tiz
 del tisunpack.exe
 
-xcopy /q TS\OGG\*.* TS-WAV\*.*
-xcopy /q TS\OGG2\*.* TS-WAV\*.*
+xcopy /q /y TS\OGG\*.* TS-WAV\*.*
+xcopy /q /y TS\OGG2\*.* TS-WAV\*.*
 
 move override\*.ogg TS-WAV
 copy TS\oggdec.exe TS-WAV\*.*
@@ -34,29 +34,29 @@
 
 xcopy /q /e /y TS\music\*.* music\*.*
 
-move override\*.ARE TS-ARE
+:: move override\*.ARE TS-ARE
 move override\*.WED TS-ARE
 weidu --make-biff TS-ARE
 call TS\deldir TS-ARE
 
-move override\*.BAM TS-BAM
+:: move override\*.BAM TS-BAM
 weidu --make-biff TS-BAM
 call TS\deldir TS-BAM
 
-move override\*.BCS TS-BCS
+:: move override\*.BCS TS-BCS
 weidu --make-biff TS-BCS
 call TS\deldir TS-BCS
 
-move override\*.CRE TS-CRE
+:: move override\*.CRE TS-CRE
 weidu --make-biff TS-CRE
 call TS\deldir TS-CRE
 
-move override\*.DLG TS-DLG
+:: move override\*.DLG TS-DLG
 weidu --make-biff TS-DLG
 call TS\deldir TS-DLG
 call TS\deldir TSDLG
 
-move override\*.ITM TS-ITM
+:: move override\*.ITM TS-ITM
 weidu --make-biff TS-ITM
 call TS\deldir TS-ITM
 
@@ -65,15 +65,15 @@
 weidu --make-biff TS-MOS
 call TS\deldir TS-MOS
 
-move override\*.2DA TS-RULE
-move override\*.CH* TS-RULE
+:: move override\*.2DA TS-RULE
+:: move override\*.CH* TS-RULE
 ::move override\*.IDS TS-RULE
-move override\*.WMP TS-RULE
-move override\*.EFF TS-RULE
-move override\*.PRO TS-RULE
-move override\*.VVC TS-RULE
-move override\*.SPL TS-RULE
-move override\*.STO TS-RULE
+:: move override\*.WMP TS-RULE
+:: move override\*.EFF TS-RULE
+:: move override\*.PRO TS-RULE
+:: move override\*.VVC TS-RULE
+:: move override\*.SPL TS-RULE
+:: move override\*.STO TS-RULE
 weidu --make-biff TS-RULE
 call TS\deldir TS-RULE
 
@@ -83,6 +83,13 @@
 move override\*.WAV TS-WAV
 weidu --make-biff TS-WAV
 call TS\deldir TS-WAV
+
+FOR %%s IN (
+override\*.baf
+override\*.d
+override\*.ogg
+override\*.tiz
+) DO del %%s
 
 @echo.
 @echo Installation complete.
