--- Setup-TS-BP.tp2	Thu Jun 21 21:24:52 2007
+++ C:\BWP Patchstudio\patched files\Setup-TS-BP.tp2	Sat Apr 17 10:35:03 2010
@@ -32,10 +32,10 @@
 
 COPY + ~Setup-TS-BP.exe~ ~WeiDU.exe~
 
-COPY + ~chitin.key~ ~TS/chitin.ts~
-COPY_LARGE + ~dialog.tlk~ ~TS/dialog.ts~
-MKDIR ~TS/over.bak~
-COPY_LARGE + ~override~ ~TS/over.bak~
+// COPY + ~chitin.key~ ~TS/chitin.ts~
+// COPY_LARGE + ~dialog.tlk~ ~TS/dialog.ts~
+// MKDIR ~TS/over.bak~
+// COPY_LARGE + ~override~ ~TS/over.bak~
 MKDIR ~TS-ARE~
 MKDIR ~TS-BCS~
 MKDIR ~TS-BAM~
