--- Setup-Vault.tp2	Mon Feb 16 08:40:37 2009
+++ C:\BWP Patchstudio\patched files\Setup-Vault.tp2	Sat Apr 17 10:36:43 2010
@@ -53,8 +53,8 @@
 
 COPY + ~Setup-Vault.exe~ ~WeiDU.exe~
 
-COPY + ~chitin.key~ ~Vault/chitin.vlt~
-COPY_LARGE + ~dialog.tlk~ ~Vault/dialog.vlt~
+// COPY + ~chitin.key~ ~Vault/chitin.vlt~
+// COPY_LARGE + ~dialog.tlk~ ~Vault/dialog.vlt~
 //MKDIR ~Vault/over.bak~
 //COPY_LARGE + ~override~ ~Vault/over.bak~
 
