--- setup-planarspheremod.tp2	Wed Mar 26 13:40:08 2008
+++ C:\BWP Patchstudio\patched files\setup-planarspheremod.tp2	Sat Apr 17 08:45:49 2010
@@ -24,10 +24,10 @@
 
 COPY + ~Setup-PlanarSphereMod.exe~ ~WeiDU.exe~
 
-COPY + ~chitin.key~ ~PlanarSphereMod/chitin.psm~
-COPY_LARGE + ~dialog.tlk~ ~PlanarSphereMod/dialog.psm~
-MKDIR ~PlanarSphereMod/over.bak~
-COPY_LARGE + ~override~ ~PlanarSphereMod/over.bak~
+// COPY + ~chitin.key~ ~PlanarSphereMod/chitin.psm~
+// COPY_LARGE + ~dialog.tlk~ ~PlanarSphereMod/dialog.psm~
+// MKDIR ~PlanarSphereMod/over.bak~
+// COPY_LARGE + ~override~ ~PlanarSphereMod/over.bak~
 
 
 // TOB Check prior to compiling.
