--- C:/Documents and Settings/Chaithanya/BiG-World-Borkpack/orig/Setup-BGT.tp2	Sun Jan 02 11:19:14 2011
+++ C:/Documents and Settings/Chaithanya/BiG-World-Borkpack/patched/Setup-BGT.tp2	Tue Jan 04 20:39:07 2011
@@ -150,13 +150,13 @@
 // chitin.key and override backup
 ///////////////////////////////////////////////////////
 
-PRINT ~~
-PRINT @100002
+// PRINT ~~
+// PRINT @100002
 
-COPY ~chitin.key~ ~chitin.key~
+// COPY ~chitin.key~ ~chitin.key~
 
-MKDIR ~BGT/over.bak~
-COPY_LARGE + ~override~ ~BGT/over.bak~
+// MKDIR ~BGT/over.bak~
+// COPY_LARGE + ~override~ ~BGT/over.bak~
 
 ///////////////////////////////////////////////////////
 // now check to prevent override directory being empty
