--- Setup-BGT.tp2	Mon Jun 13 12:02:50 2011
+++ C:\BWP Patchstudio\patched files\Setup-BGT.tp2	Wed Oct 26 22:49:39 2011
@@ -153,13 +153,13 @@
 // chitin.key and override backup
 ///////////////////////////////////////////////////////
 
-PRINT ~~
-PRINT @100002
+// PRINT ~~
+// PRINT @100002
 
-COPY ~chitin.key~ ~chitin.key~
+// COPY ~chitin.key~ ~chitin.key~
 
-MKDIR ~BGT/over.bak~
-COPY_LARGE + ~override~ ~BGT/over.bak~
+// MKDIR ~BGT/over.bak~
+// COPY_LARGE + ~override~ ~BGT/over.bak~
 
 ///////////////////////////////////////////////////////
 // now check to prevent override directory being empty
