--- Setup-CtB.tp2	Fri Feb 04 19:00:14 2011
+++ C:\BWP Patchstudio\patched files\Setup-CtB.tp2	Wed Oct 26 22:49:51 2011
@@ -37,10 +37,10 @@
 
 COPY + ~Setup-CtB.exe~ ~WeiDU.exe~
 COPY + ~Setup-CtB.exe~ ~Setup-CtB-Chores.exe~
-COPY + ~chitin.key~ ~CtB/chitin.ctb~
-COPY_LARGE + ~dialog.tlk~ ~CtB/dialog.ctb~
-MKDIR ~CtB/over.bak~
-COPY_LARGE + ~override~ ~CtB/over.bak~
+// COPY + ~chitin.key~ ~CtB/chitin.ctb~
+// COPY_LARGE + ~dialog.tlk~ ~CtB/dialog.ctb~
+// MKDIR ~CtB/over.bak~
+// COPY_LARGE + ~override~ ~CtB/over.bak~
 MKDIR ~CTB-ARE~
 MKDIR ~CTB-BCS~
 MKDIR ~CTB-BAM~
