--- ToA\setup-toa.TP2	Fri Jan 06 20:49:24 2012
+++ C:\BWP Patchstudio\patched files\ToA\setup-toa.TP2	Wed Sep 12 00:00:05 2012
@@ -22,18 +22,18 @@
          ~ToA/language/italian/journal.tra~
 
 BEGIN @1 DESIGNATED 01
-MKDIR ~ToA/over.bak~
-COPY_EXISTING + ~AR0700.bcs~    ~ToA/over.bak~
-COPY_EXISTING + ~AR5500.bcs~    ~ToA/over.bak~
+// MKDIR ~ToA/over.bak~
+// COPY_EXISTING + ~AR0700.bcs~    ~ToA/over.bak~
+// COPY_EXISTING + ~AR5500.bcs~    ~ToA/over.bak~
 
-COPY + ~setup-toa.exe~ ~Weidu.exe~
-COPY + ~chitin.key~ ~ToA/chitin.toa~
-COPY_LARGE + ~dialog.tlk~  ~ToA/dialog.toa~
+// COPY + ~setup-toa.exe~ ~Weidu.exe~
+// COPY + ~chitin.key~ ~ToA/chitin.toa~
+// COPY_LARGE + ~dialog.tlk~  ~ToA/dialog.toa~
 
-ACTION_IF FILE_EXISTS ~dialogf.tlk~
-THEN BEGIN
-COPY_LARGE + ~dialogf.tlk~  ~ToA/dialogf.toa~
-END
+// ACTION_IF FILE_EXISTS ~dialogf.tlk~
+// THEN BEGIN
+// COPY_LARGE + ~dialogf.tlk~  ~ToA/dialogf.toa~
+// END
 
 EXTEND_BOTTOM ~AR0700.bcs~ ~ToA/areabaf/AR0700.baf~
 EXTEND_BOTTOM ~AR5500.bcs~ ~ToA/areabaf/AR5500.baf~
