--- Setup-NTotSC.TP2	Thu Jan 01 15:33:30 2009
+++ C:\BWP Patchstudio\patched files\Setup-NTotSC.TP2	Sun Aug 25 10:19:39 2013
@@ -47,12 +47,12 @@
 
 BEGIN @0
 
-MKDIR ~NTotSC/over.bak~
-COPY_LARGE + ~override~    ~NTotSC/over.bak~
+// MKDIR ~NTotSC/over.bak~
+// COPY_LARGE + ~override~    ~NTotSC/over.bak~
 
-COPY + ~Setup-NTotSC.exe~  ~WeiDU.exe~
-COPY + ~chitin.key~        ~NTotSC/chitin.nts~
-COPY_LARGE + ~dialog.tlk~  ~NTotSC/dialog.nts~
+// COPY + ~Setup-NTotSC.exe~  ~WeiDU.exe~
+// COPY + ~chitin.key~        ~NTotSC/chitin.nts~
+// COPY_LARGE + ~dialog.tlk~  ~NTotSC/dialog.nts~
 
 MKDIR ~NTOTSC1~
 MKDIR ~NTOTSC2~
