--- Setup-BoneHillv275.tp2	Sun Oct 02 04:24:17 2011
+++ C:\BWP Patchstudio\patched files\Setup-BoneHillv275.tp2	Wed Oct 26 22:50:50 2011
@@ -41,12 +41,12 @@
 
 BEGIN @0
 
-MKDIR ~bonehillv275/over.bak~
-COPY_LARGE + ~override~      ~bonehillv275/over.bak~
+// MKDIR ~bonehillv275/over.bak~
+// COPY_LARGE + ~override~      ~bonehillv275/over.bak~
 
-COPY + ~Setup-BoneHillv275.exe~  ~WeiDU.exe~
-COPY + ~chitin.key~        ~bonehillv275/chitin.bh~
-COPY_LARGE + ~dialog.tlk~  ~bonehillv275/dialog.bh~
+// COPY + ~Setup-BoneHillv275.exe~  ~WeiDU.exe~
+// COPY + ~chitin.key~        ~bonehillv275/chitin.bh~
+// COPY_LARGE + ~dialog.tlk~  ~bonehillv275/dialog.bh~
 
 MKDIR ~BH-TIS1~
 MKDIR ~BH-TIS2~
@@ -1722,10 +1722,10 @@
 
 ACTION_IF FILE_EXISTS ~data/BG1ARE.bif~
  THEN BEGIN
-  COPY	~Bonehillv275/creatures/spawns~	~BH-REST~
+  COPY	~Bonehillv275/creatures/spawns~	~override~
   COMPILE	~Bonehillv275/BGT/spawns~
  END ELSE BEGIN
-  COPY	~Bonehillv275/creatures/spawns~	~BH-REST~
+  COPY	~Bonehillv275/creatures/spawns~	~override~
   COMPILE	~Bonehillv275/Tutu/spawns~
 END
 
